hail/python/hail/methods/qc.py

```diff
@@ -492,13 +512,548 @@ def n_discordant(counter):
+{vcf_or_json} \
+--everything \
+--allele_number \
+--no_stats \
+--cache \
+--offline \
+--minimal \
+--assembly GRCh37 \
+--dir={self.data_mount} \
+--plugin LoF,human_ancestor_fa:{self.data_mount}/loftee_data/human_ancestor.fa.gz,filter_position:0.05,min_intron_size:15,conservation_file:{self.data_mount}/loftee_data/phylocsf_gerp.sql,gerp_file:{self.data_mount}/loftee_data/GERP_scores.final.sorted.txt.gz \
+-o STDOUT
+'''
+
+
+class HailVEPConfig_GRCh38_95(HailVEPConfig):
+    """
+    The Hail-maintained VEP configuration for GRCh38 for VEP version 95.
+
+    This class takes the following constructor arguments:
+
+     - `data_bucket` (:obj:`.str`) -- The location where the VEP data is stored.
+     - `data_mount` (:obj:`.str`) -- The location in the container where the data should be mounted.
+     - `image` (:obj:`.str`) -- The docker image to run VEP.
+     - `cloud` (:obj:`.str`) -- The cloud where the Batch Service is located.
+     - `reference_data_is_bucket_requester_pays` (:obj:`.bool`) -- True if the reference data is in a requester pays bucket.
+     - `regions` (:obj:`.list` of :obj:`.str`) -- A list of regions the VEP jobs can run in.
+                 data_bucket: str,
+                 regions: List[str],
+                 image: str,
+                 data_mount: str,
+                 cloud: Optional[str]):
+
+    """
+
+    vep_json_typ = HailVEPConfig.vep_json_typ._insert_field('transcript_consequences', hl.tarray(
+        HailVEPConfig.vep_json_typ['transcript_consequences'].element_type._insert_fields(
+            appris=hl.tstr,
+            tsl=hl.tint32,
+        )
+    ))
+
+    def vep_command(self,
+                    *,
+                    consequence: bool,
+                    tolerate_parse_error: bool,
+                    part_id: int,
+                    input_file: Optional[str],
+                    output_file: str) -> str:
+        vcf_or_json = '--vcf' if consequence else '--json'
+        input_file = f'--input_file {input_file}' if input_file else ''
+        return f'''/vep/vep {input_file} \
+--format vcf \
+{vcf_or_json} \
+--everything \
+--allele_number \
+--no_stats \
+--cache \
+--offline \
+--minimal \
+--assembly GRCh38 \
+--fasta {self.data_mount}/homo_sapiens/95_GRCh38/Homo_sapiens.GRCh38.dna.toplevel.fa.gz \
+--plugin "LoF,loftee_path:/vep/ensembl-vep/Plugins/,gerp_bigwig:{self.data_mount}/gerp_conservation_scores.homo_sapiens.GRCh38.bw,human_ancestor_fa:{self.data_mount}/human_ancestor.fa.gz,conservation_file:{self.data_mount}/loftee.sql" \
+--dir_plugins /vep/ensembl-vep/Plugins/ \
+--dir_cache {self.data_mount} \
+-o STDOUT
+'''
+
+
+supported_vep_configs = {
+    ('GRCh37', 'gcp', 'us-central1', 'hail.is'): HailVEPConfig_GRCh37_85(
+        data_bucket='hail-qob-vep-grch37-us-central1',
+        data_mount='/vep_data/',
+        image=HAIL_GENETICS_VEP_GRCH37_85_IMAGE,
+        regions=['us-central1'],
+        cloud='gcp',
+        reference_data_is_bucket_requester_pays=True,
```

Comment:
I think you want the `is` after `bucket`, yeah?

Reply:
Should it just be `data_bucket_is_requester_pays`? That's the name of the relevant config parameter.